### PR TITLE
I56 initial value as a variable - update informative section

### PR DESCRIPTION
**Pull request by kerimoyle**
Addresses #56 

I've updated the "see more" section of the C.6 page with an example model.  Thoughts?  I'm not convinced what I've said is correct, but it's a place to start from. ...   The rendered version is here:
https://cellml-specification-dev.readthedocs.io/en/i56_initial_variables/reference/formal_and_informative/specC06_interpretation_of_initial_values.html

### CODE REVIEW

File: reference/informative/informC06_interpretation_of_initial_values.rst [hunk 1]
```diff
@@ -8,4 +8,117 @@
 
   .. container:: infospec
 
-    **TODO**
\ No newline at end of file
+    Best practice suggests that the :code:`initial_value` attribute is used to set the initial values only, rather than to simply set the value of a :code:`variable`.  
+    Initialising a variable is different from setting the value of a variable; the former is true only at the *beginning* of a simulation, whereas the latter is true *throughout* the simulation.
+    The option to use a variable reference for initialisation is provided in CellML 2.0 to enable easier migration from CellML 1.1 models, and may not be supported in future releases.
```
Comment (MichaelClerx) [inline reference/informative/informC06_interpretation_of_initial_values.rst]: Thanks! This is looking good but there's quite a lot going on here!
Maybe we could separate it into a few key points, in different paragraphs (possibly with headings?)

- semantics intial value true at t0 vs equation true always
- init used to set constants is not recommended but possible
- init can now be on connected var, allowing init vals to be set in a separate component (possibly imported)
  - init = var can be used to set init vals in separate component (possibly imported) but not recommended

Comment (kerimoyle) [reply reference/informative/informC06_interpretation_of_initial_values.rst]: Thanks :) I don't quite understand how to put this bit into practice:

> init can now be on connected var, allowing init vals to be set in a separate component (possibly imported)
> init = var can be used to set init vals in separate component (possibly imported) but not recommended

So ... init = val in external component, no init through the rest of the connected set? So my "good" example is not really so good ... how should I change it?

**Comment by kerimoyle** [comment]
Included info on #58 here too, as it's in the same block.  Hopefully example is correct and wording clearer, but please suggest if not ... :)

File: reference/informative/informC06_interpretation_of_initial_values.rst (hunk 1 above)
Comment (MichaelClerx) [reply reference/informative/informC06_interpretation_of_initial_values.rst]: I'd just do an ODE with a component
```
component clever_stuff
    variable x
    variable y
    math:
        dx/dt = ...
        dy/dy = ...
component initialisation
    variable x initial_value=3
    variable y initial_value=7
```
?

File: reference/informative/informC06_interpretation_of_initial_values.rst [hunk 2]
```diff
@@ -8,4 +8,148 @@
...
+
+    - A :code:`variable` whose value needs to be set for only the *beginning* of a simulation is initialised using the :code:`initial_value` attribute. 
+      
+        - This is most frequently used for state variables (those whose value is found by solving a differential equation).
+        - It's possible - but not recommended - to use a variable reference with the :code:`initial_value` attribute.
+          This option remains in CellML 2.0 only to provide ease of migration from CellML 1.1 models, and may be discontinued in future versions.
+          The recommended best practice is to make use of connections between variables to move all "hardcoding" of parameters (including initial values and constants) into a top-level component for ease of access.
+          This is demonstrated in the example below. 
+      
+    - A :code:`variable` whose value is *constant* throughout the simulation is best set within the :code:`math` block, rather than via the :code:`initial_value` attribute.  This is so that its value will be held to be true *throughout* the simulation, rather than only the beginning.
```
Comment (MichaelClerx) [inline reference/informative/informC06_interpretation_of_initial_values.rst]: New line!

File: reference/informative/informC06_interpretation_of_initial_values.rst [hunk 3]
```diff
@@ -8,4 +8,148 @@
...
+
+        Best practice for constants, variables, and initial conditions
+    
+
+    - A :code:`variable` whose value needs to be set for only the *beginning* of a simulation is initialised using the :code:`initial_value` attribute. 
+      
+        - This is most frequently used for state variables (those whose value is found by solving a differential equation).
+        - It's possible - but not recommended - to use a variable reference with the :code:`initial_value` attribute.
+          This option remains in CellML 2.0 only to provide ease of migration from CellML 1.1 models, and may be discontinued in future versions.
+          The recommended best practice is to make use of connections between variables to move all "hardcoding" of parameters (including initial values and constants) into a top-level component for ease of access.
```
Comment (MichaelClerx) [inline reference/informative/informC06_interpretation_of_initial_values.rst]: Maybe we can just leave this sentence out? I don't think it's necessarily best practice to do this, just that _if you want to have them in a separate component_, then this is the best way to do it

File: reference/informative/informC06_interpretation_of_initial_values.rst (hunk 1 above)
Comment (kerimoyle) [reply reference/informative/informC06_interpretation_of_initial_values.rst]: Yep, got it, thanks.

File: reference/informative/informC06_interpretation_of_initial_values.rst (hunk 3 above)
Comment (kerimoyle) [reply reference/informative/informC06_interpretation_of_initial_values.rst]: OK, will do.

Comment (kerimoyle) [reply reference/informative/informC06_interpretation_of_initial_values.rst]: Done in 6a718c4

File: reference/informative/informC06_interpretation_of_initial_values.rst (hunk 2 above)
Comment (kerimoyle) [reply reference/informative/informC06_interpretation_of_initial_values.rst]: Done in 6a718c4

File: reference/sectionB_elements.inc [hunk 4]
```diff
@@ -359,7 +359,7 @@ A :code:`math` element information item (referred to in this specification as a
 
 .. container:: issue-math-cn-units-attribute
 
-   4. Any MathML :code:`cn` elements MUST each have an attribute in the :ref:`CellML namespace<specA_cellml_namespace>`, with a local name equal to :code:`units`. The value of this attribute MUST be a valid units reference.
+   4. Any MathML :code:`cn` elements MUST each have an attribute in the :ref:`CellML namespace<specA_cellml_namespace>`, with a local name equal to :code:`units`. The value of this attribute MUST be a valid :ref:`units reference<specC_units_reference>`.
```
Comment (agarny) [inline reference/sectionB_elements.inc]: Two sentences, so two lines.

Comment (kerimoyle) [reply reference/sectionB_elements.inc]: Done in 770e756

**Pull request merged**